Stripe/StripeError.m

```diff
@@ -148,3 +148,15 @@ + (nonnull NSString *)stp_cardErrorProcessingErrorUserMessage {
 }
 
 @end
+
+@implementation NSError (STPCustomerContext)
+
++ (NSError *)stp_customerContextMissingKeyProviderError {
```

Comment:
👍 , I'll move it. Realizing we should probably also be doing the `linkCategoryForStaticLib` thing for this and `StripeError.m` – I'll add.